napari/components/_viewer/view/main.py

```diff
@@ -9,6 +11,13 @@
 from .controls import QtControls
 from .buttons import QtLayersButtons
 
+import os.path as osp
+from ....resources import resources_dir
+from ....util.theme import template, palettes
```

Comment:
I this `theme` stuff needed for the `keybindings` PR, or is it compensating for other refactoring, or can it be removed till later?

Reply:
That must've been my mistake when rebasing, will fix.

Reply:
This doesn't seem fixed yet. =P

Also, the dreaded quadruple-dot has reared its ugly head. Perhaps this is not the PR to fix it but just want to flag again that I think our hierarchy is too deep and complex. `components` seems redundant, actually... Isn't everything a component?

Reply:
(Note, please don't fix this in this PR, it'll be a mess! :joy:)